src/components/entry/grid-entry.tsx

```diff
@@ -1,20 +1,44 @@
-import {Entry, EntryProps} from './entry';
-import {EntryText} from './entry-text';
+import {ComponentChildren} from 'preact';
+import {useContext} from 'preact/hooks';
+
+import {MultilineText} from 'components/multiline-text/multiline-text';
+import {RelatedContext} from 'components/related-context/related-context';
+import {EntryImage} from 'components/entry-image/entry-image';
+
 import * as styles from './entry.scss';
+import {EntryDimensions} from 'types/entry-dimensions';
 
-interface GridEntryProps extends EntryProps {
+interface GridEntryProps {
   title?: string;
+  id: number;
+  children?: ComponentChildren;
+  duration?: number;
+  type: KalturaPlayerTypes.EntryTypes;
+  poster?: string;
+  entryDimensions: EntryDimensions;
 }
 
 const GridEntry = (props: GridEntryProps) => {
-  const {title, ...otherProps} = props;
+  const {relatedManager} = useContext(RelatedContext);
+
+  const {id, title, duration, type, poster, entryDimensions} = props;
+  const {width, imageHeight, contentHeight} = entryDimensions;
+
   return (
     <div className={styles.gridEntry}>
-      <Entry {...otherProps}>
-        <div className={styles.text}>
-          <EntryText text={title || ''} />
+      <div
+        className={`${styles.entry} ${styles.clickable}`}
```

Comment:
`styles.gridEntry` can be moved to here like in minimal? and what's about `clickable` in minimal?